proof_test.go

```diff
@@ -1125,3 +1125,56 @@ func TestProofOfPresenceWithEmptyValue(t *testing.T) {
 		t.Fatal("differing commitment for child #0")
 	}
 }
+
+func TestDoubleProofOfAbsence(t *testing.T) {
```

Comment:
I'm adding yet another case here.
The idea is to generate a proof of two keys (with different stems!) that end up in the same LeafNode.